packages/notifications/src/styled/content/StyledClose.tsx

```diff
@@ -6,29 +6,61 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import CalloutStyles from '@zendeskgarden/css-callouts';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
+import { retrieveComponentStyles, getColor } from '@zendeskgarden/react-theming';
+import { VALIDATION_HUE } from '../../utils/types';
 
 const COMPONENT_ID = 'notifications.close';
 
-export interface IStyledCloseProps {
-  isFocused?: boolean;
-  isHovered?: boolean;
+interface IStyledCloseProps {
+  hue?: VALIDATION_HUE;
 }
 
 /**
  * Used to close a Notification. Supports all `<button>` props
+ *
+ * 1. Reset for <button> element.
  */
-export const StyledClose = styled.button.attrs<IStyledCloseProps>(props => ({
+export const StyledClose = styled.button.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  type: 'button',
-  className: classNames(CalloutStyles['c-callout__close'], {
-    // State
-    [CalloutStyles['is-focused']]: props.isFocused,
-    [CalloutStyles['is-hovered']]: props.isHovered
-  })
-}))<IStyledCloseProps>`
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledCloseProps>`
+  display: block;
+  position: absolute;
+  top: ${props => props.theme.space.base}px;
+  ${props => (props.theme.rtl ? 'left' : 'right')}: ${props => `${props.theme.space.base}px`};
+  transition: background-color 0.1s ease-in-out;
+  border: none; /* [1] */
+  border-radius: 50%;
+  background-color: transparent; /* [1] */
+  cursor: pointer;
+  padding: 0;
+  width: ${props => props.theme.space.base * 7}px;
+  height: ${props => props.theme.space.base * 7}px;
+  overflow: hidden;
+  color: ${props =>
+    props.hue
+      ? getColor(props.hue, props.hue === 'warningHue' ? 700 : 500, props.theme)
+      : getColor('neutralHue', 600, props.theme)};
```

Comment:
Color for the close icon is the `600` shade (unless `'warningHue'` which is 700). I think the CSS got it wrong compared to the current designs.

Reply:
Yeah that's correct. My mistake! This should be fixed with the latest commit.